src/SourceBuild/content/build.sh

```diff
@@ -365,7 +365,7 @@ if [[ "$sourceOnly" == "true" ]]; then
     packageVersionsPath="$CUSTOM_PACKAGES_DIR/PackageVersions.props"
   elif [ -d "$packagesArchiveDir" ]; then
     sourceBuiltArchive=$(find "$packagesArchiveDir" -maxdepth 1 -name 'Private.SourceBuilt.Artifacts*.tar.gz')
-    if [ -f "${packagesPreviouslySourceBuiltDir}}PackageVersions.props" ]; then
+    if [ -f "${packagesPreviouslySourceBuiltDir}PackageVersions.props" ]; then
```

Comment:
This typo should probably be fixed anyway, even if the rest of the PR isn't merged.

Reply:
I opened a separate PR for this: https://github.com/dotnet/installer/pull/19619